src/main/scala/rifs/business/tables/OpportunityTables.scala

```diff
@@ -110,6 +110,15 @@ class OpportunityTables @Inject()(val dbConfigProvider: DatabaseConfigProvider,
   lazy val sectionC = Compiled(sectionQ _)
 
   override def saveSectionDescription(id: OpportunityId, sectionNo: Int, description: Option[String]): Future[Int] = db.run {
-    sectionQ(id, sectionNo).map(r=> r.text).update(description)
+    sectionQ(id, sectionNo).map(r => r.text).update(description)
+  }
+
+  override def reset(): Future[Unit] = {
```

Comment:
Does this mean that testing is dependant on particular ID numbers?
I'd say that resetting sequence is not normally needed